docs/roadmap/ARCHIVE.md

```
@@ -0,0 +1,183 @@
+- [ ] UI reworks
+    - [x] Redesign Dashboard
+    - [x] Redesign Pre-match simulation screen
+    - [x] Redesign Post-match simulation screen
+    - [x] Redesign tactics board
+    - [x] Redesign match
+    - [ ] ~Redesign training screen~ (Deferred to 0.3.1-beta)
+    - [x] Redesign loan and transfers screen
+    - [x] Redesign the Schedule screen
+- [x] Support loan bids
+
+In this release, the user must be able to choose the region and team that they want to play. The championships are now expanded, so you can play more than one cup at the same time.
```

markdownlint-cli2 (0.23.0)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)